classes/SlackConnector.class.js

```diff
@@ -57,14 +65,26 @@ SlackConnector.prototype = {
 		    this.text = message.text,
 		    this.response = '';
 			slashbot.message(this.user.name, this.text);
+
+			if(that.activeUsersArray.indexOf(this.user.name) == -1){
+				that.activeUsersArray.push(this.user.name);
+			}
+
+			that.slashbot.registerPlayers(that.activeUsersArray);
+
+		});
+
+		slack.on('error', function(error) {
+			console.error('Error: %s', error);
 		});
 
 		slack.login();
 		this.slack = slack;
 	},
 	say: function(who, text){
+		console.log(typeof who);
 		console.log("Saying: " + text + " to " + who);
-		var dm = this.slack.getChannelGroupOrDMByName(who.name);
+		var dm = this.slack.getChannelGroupOrDMByName(who);
```

Comment:
I've thinking on this, following the duck type convention we might support both strings and objects which have name. (Note, not necessarily "user" objects).   For now, sending the name directly should do.